README.md

```diff
@@ -1,4 +1,4 @@
-[![Coverage Status](https://coveralls.io/repos/github/andela-moseni/inverted-index/badge.svg?branch=development)](https://coveralls.io/github/andela-moseni/inverted-index?branch=development) [![Build Status](https://travis-ci.org/andela-moseni/inverted-index.svg?branch=development)](https://travis-ci.org/andela-moseni/inverted-index)
+[![Code Climate](https://codeclimate.com/github/andela-moseni/inverted-index/badges/gpa.svg)](https://codeclimate.com/github/andela-moseni/inverted-index) [![Coverage Status](https://coveralls.io/repos/github/andela-moseni/inverted-index/badge.png?branch=development)](https://coveralls.io/github/andela-moseni/inverted-index?branch=development) [![Build Status](https://travis-ci.org/andela-moseni/inverted-index.svg?branch=development)](https://travis-ci.org/andela-moseni/inverted-index)
 # Inverted Index
 ## Introduction
 An Inverted Index is an Index Data Structure storing a mapping from content to location (i.e. a mapping of words or numbers to it's location in a database file). It use the elasticsearch method, which is designed to allow very fast full-text searches.
```

Comment:
Good job! You should also include the link to your hosted application, so that users can access the application.